lib/ruby_home/config/characteristics.yml

```diff
@@ -30,6 +30,18 @@
     ValidValues:
       '0': Inactive
       '1': Active
+- :name: :active_identifier
+  :description: Active Identifier
+  :uuid: 000000E7-0000-1000-8000-0026BB765291
+  :format: uint32
+  :unit: nil
+  :permissions:
+  - securedRead
+  :properties:
+  - read
+  - write
+  - cnotify
```

Comment:
I noticed that most things are ordered alphabetically, but not the `properties` field. Is this ordering correct?

Reply:
Any order is fine. This file is normally automatically generated from https://github.com/karlentwistle/ruby_home/blob/master/sbin/characteristic_generator.rb